### ISSUE TITLE
the link"https:/github.com/hutchresearch/safekit.git"is lost?

### ISSUE THREAD

**Issue by ZhcccdNancy** [pnnl/safekit#1]
hello,I want to run this program on Win10 ,tensorflow 1.0+.But link https:/github.com/hutchresearch/safekit.git   I  could not find.So where can I download "hutchresearch/safekit"
thank you~

**Comment by aarontuor** [comment]
The hutchresearch repo is private. The publicly released code is at:

github.com/pnnl/safekit

Where did you get the hutchresearch link?

Thanks,
Aaron

**Comment by ZhcccdNancy** [comment]
Thanks for your reply! problem is solved!
I want to run this program on Windows,but error exists that many paths in the program are not found,like ：
python: can't open file '/safekit/models/dnn_agg.py': [Errno 2] No such file or directory
cert dnn autoencoder w/diag covariance, batch norm, input norm, replay

I find the difference of system path between linux and wndows.windows uses '' and linux uses '/'.
I change '/' for '\' in agg_test.py and lanl_lm_test.py ,but problems still exist.
Shall I change all paths in this program?
thank you ~

**Comment by aarontuor** [comment]
Yes. You will probably have to change any of these relative paths that are in the testing scripts. The code was written for a linux system so there may be some issues running on windows.
